### DISCUSSION TITLE
Code block search hotkey to select different code formats

### DISCUSSION THREAD

**Discussion by jonboii** [GitbookIO/community#455] [Feature Requests]
Title basically sums it up.  I know this is extremely lazy, but it would amazing to have a new hotkey.  Much like `/` opens the search for an `insert block` on a new line, it would be a separate hotkey maybe like `CTRL + k` (for kode) after selecting a code block. This way writing a new line after a paragraph would go as follows:

`/`
Code Block
`Enter`
`Ctrl + k`
Markdown
`Enter`
<start typing out my code hur>

Again, super lazy, but then we wouldn't have to take our hands off the keyboard just to select a different code format.  Love what you all do, keep up the good work!! kthxbye

**Comment by SamyPesse** [comment]
Thanks for the kind words :)

You should be able to use only the keyboard to insert a code block with either:

Using the markdown shortcut to insert the code block with a specific syntax, type the following, then press <kbd>Enter</kbd>:

    ```js

(You can replace `js` with another language; most should be recognized).

Or use the keyboard shortcut <kbd>Command+/</kbd> (on Mac) or <kbd>Ctrl+/</kbd> (on Windows/Linux) to open the menu to edit the block, then you can press <kbd>Escape</kbd> to focus back the content of the code block.

**Reply by jonboii** [reply]
wow, if I could insert the Pikachu meme as text art I would.  Went right over my head when reading the docs, Thank you!
